template/1-overview.md

```diff
@@ -3,8 +3,16 @@
 *Maximum: 2 pages*
 
 *What is this project about?*
-
 *Describe succinctly and precisely its purpose.*
-
 *Include a 3-sentence (max) elevator pitch.*
 
+Although people get closer through social media, they are physically farther than ever, so we present you Proxima.
+
+Imagine a social app that encourages people to visit interesting places instead of staying at home. allows users to create and share posts with a unique twist: to read a post, users must physically visit the location where it was published.
```

Comment:
I believe there is a missing word here before `allows`:
```suggestion
Imagine a social app that encourages people to visit interesting places instead of staying at home. This allows users to create and share posts with a unique twist: to read a post, users must physically visit the location where it was published.
```

Reply:
Thanks for the catch! Fixed in edb5ab6e346f02fb0e98ce1c4086a86d067c121e